wdl-lint/tests/lints/command-mixed-line-cont/source.errors

```diff
@@ -1,3 +1,11 @@
+note[DescriptionMissing]: description key missing in meta section
+  ┌─ tests/lints/command-mixed-line-cont/source.wdl:7:5
+  │
+7 │     meta {}
```

Comment:
For all of these, I would add `#@ except: DescriptionMissing` to the top of the document so that these errors don't show up for non-relevant tests.